business-directory-plugin.php

```diff
@@ -121,6 +121,18 @@ function wpbdp() {
 	return $instance;
 }
 
+// Increase the priority value for AddToAny widgets.
```

Comment:
Can we move this to a different file to avoid bulking this one up? This looks like a good spot for it:
https://github.com/Strategy11/business-directory-plugin/blob/master/includes/compatibility/class-compat.php

Reply:
Done, I moved it into a function there. But the one for SSB has to stay in the main plugin file, otherwise, the filter will be executed before we can increase its value.